docs/source/relayer.rst

```diff
@@ -4,6 +4,13 @@ Relayer
 Starting relayer
 ----------------
 
+There are two ways to setup relayer :ref:`from command line<from-command-line>` and :ref:`programmatically<programmatically>`.
+
+If you would like to have your own domain, jump to the section :ref:`ENS registration<ens-registration>`
+To build custom relayer jump to :ref:`Custom relayer<custom-relayer>`
+
+.. _from-command-line:
+
 From command line
 ^^^^^^^^^^^^^^^^^
 
```

Comment:
Line below: clone github [Universal Login repositry]() and follow the steps:

Reply:
In steps below:
add titles: Setup environment, Run relayer
Run following command from `universal-login-relayer` directory